owners/OWNERS.example.yaml

```diff
@@ -58,17 +59,40 @@
 - "*.{js,css}": frontend
 - "package.{,lock.}json": somebodyelse
 
+# Prepending a user or team name with a `?` will prevent this owner from being
```

Comment:
Line 20: Might be useful to provide a link to the list of GH teams for `ampproject/amphtml`.

Reply:
Added.